clippy_utils/src/check_proc_macro.rs

```diff
@@ -319,6 +319,44 @@ fn attr_search_pat(attr: &Attribute) -> (Pat, Pat) {
     }
 }
 
+// TODO: Waiting on `ty_search_pat`.
+// fn where_pred_search_pat(where_pred: &WherePredicate<'_>) -> (Pat, Pat) {
+//     match where_pred {
+//         WherePredicate::BoundPredicate(bound) => {
+//             todo!();
+//         },
+//         WherePredicate::RegionPredicate(region) => {
+//
+//         },
+//         WherePredicate::EqPredicate(..) => unimplemented!(),
+//     }
+// }
```

Comment:
Should this comment be part of this PR or is it just an artifact of your dev process? :D

Reply:
Forgot to finish that, I think I'll remove it since checking the `Ty` already works